ecmascript.sublime-syntax

```diff
@@ -76,7 +76,7 @@ variables:
       (?<![{{identifierContinueChars}}\\])
       {{identifierAllCaps}}
   identifierName: '{{identifierStart}}{{identifierPart}}*'
-  identifierContinueChars: '\p{IDS}$\x{200C}\x{200D}'
+  identifierContinueChars: '[\p{IDS}$_]$\x{200C}\x{200D}'
```

Comment:
```suggestion
  identifierContinueChars: '\p{IDC}$$\x{200C}\x{200D}'
```

Oops!  The fix is that this should be IDC, not IDS. IDC is inclusive of underscore. This production is used within [] rather than including it.